content/en/agent/guide/python-3.md

```diff
@@ -9,11 +9,38 @@ This guide provides information and best practices on migrating checks between P
 
 To provide flexibility in allowing code to run multiple on versions of the Agent, this guide focuses on retaining backwards compatibility.
 
+## Agent configuration
+
+Starting with v6.14.0, the Agent integrates both Python 2 and Python 3 runtimes. This means that custom Agent Checks can be run either with Python 2 or Python 3, depending on the Agent configuration.
```

Comment:
I'd remove `custom` here, since it applies to both custom and official checks
```suggestion
Starting with v6.14.0, the Agent integrates both Python 2 and Python 3 runtimes. This means that Agent Checks can be run either with Python 2 or Python 3, depending on the Agent configuration.
```